doc/release/1.11.0-notes.rst

```diff
@@ -25,9 +32,19 @@ Build System Changes
 Future Changes
 ==============
 
-* Relaxed stride checking will become the default in 1.12.0.
-* Support for Python 2.6, 3.2, and 3.3 will be dropped in 1.12.0.
-* ``MaskedArray`` takes view of data **and** mask when slicing in 1.12.0.
+The following changes are scheduled for Numpy 1.12.0.
+
+* Relaxed stride checking will become the default.
```

Comment:
It *is* opaque ;) The 1.8.0 release notes has a discussion and there is a brief mention in the 'Internal memory layout of an ndarray' section of the reference guide.